presto-hive/src/main/java/io/prestosql/plugin/hive/security/SqlStandardAccessControl.java

```diff
@@ -263,6 +264,14 @@ public void checkCanRenameColumn(ConnectorSecurityContext context, SchemaTableNa
         }
     }
 
+    @Override
+    public void checkCanSetTableAuthorization(ConnectorSecurityContext context, SchemaTableName tableName, PrestoPrincipal principal)
+    {
+        if (!isTableOwner(context, tableName)) {
```

Comment:
Is this a Hive behavior? What about being an owner of a schema that contains that table, would that allow to change the owner of table?

Reply:
The ALTER TABLE SET AUTHORIZATION syntax is not a part of SQL and not a part of Hive. The only way to change table's owner is via metastore

Reply:
@dain @electrum What do you think about this?

Reply:
I thought about it, and I think it is ok.